WordPressVIPMinimum/Sniffs/VIP/RestrictedVariablesSniff.php

```diff
@@ -46,6 +46,13 @@ public function getGroups() {
 					'$wpdb->usermeta',
 				],
 			],
+			'session' => [
+				'type'      => 'error',
+				'message'   => 'Due to using Batcache, server side based client related logic will not work, use JS instead.',
```

Comment:
Was this message a copy and paste from other messages, or is it equally valid for this?

The previous message was simply `'Usage of $_SESSION variable is prohibited.'`